### PR TITLE
[14.0[MIG]website_crm_quick_answer, standard migration

### PR DESCRIPTION
**Pull request by manuelregidor**
Supersedes https://github.com/OCA/website/pull/798

### CODE REVIEW

**Comment by manuelregidor** [comment]
@HaraldPanten @ValentinVinagre

**Review by HaraldPanten — APPROVED** [review]
Functional review 👍

File: website_crm_quick_answer/README.rst [hunk 1]
```diff
@@ -14,13 +14,13 @@ Quick answer for website contact form
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
 .. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fwebsite-lightgray.png?logo=github
-    :target: https://github.com/OCA/website/tree/12.0/website_crm_quick_answer
+    :target: https://github.com/OCA/website/tree/13.0/website_crm_quick_answer
```
Comment (gdgellatly) [inline website_crm_quick_answer/README.rst]: all this page, 14.0

**Review by xavigutipujol — APPROVED** [review]
LGTM 👍

File: website_crm_quick_answer/data/base_automation_data.xml [hunk 2]
```diff
@@ -22,22 +22,15 @@
...
+        </field>
+    </record>
+    <record id="automated_action" model="base.automation">
+        <field name="name">Quick response to website contact form</field>
+        <field name="model_id" ref="crm.model_crm_lead" />
+        <field name="state">email</field>
+        <field name="trigger">on_create</field>
+        <field name="active" eval="True" />
+        <field name="template_id" ref="email_template" />
+        <field name="filter_domain">[["medium_id.name","=","Website"]]</field>
```
Comment (pedrobaeza) [inline website_crm_quick_answer/data/base_automation_data.xml]: This doesn't serve on translated terms. Have you tried to put `ref()` here to see if it works?

**Comment by HaraldPanten** [comment]
@gdgellatly changes done

File: website_crm_quick_answer/data/base_automation_data.xml (hunk 2 above)
Comment (manuelregidor) [reply website_crm_quick_answer/data/base_automation_data.xml]: It works. Change added.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**